library/oneview_storage_pool.py

```diff
@@ -75,15 +97,21 @@
 
 
 from ansible.module_utils.basic import AnsibleModule
-from module_utils.oneview import OneViewModuleBase, HPOneViewValueError
+from module_utils.oneview import OneViewModuleBase, HPOneViewValueError, HPOneViewResourceNotFound, ResourceComparator
 
 
 class StoragePoolModule(OneViewModuleBase):
     MSG_CREATED = 'Storage Pool added successfully.'
     MSG_ALREADY_PRESENT = 'Storage Pool is already present.'
+    MSG_UPDATED = 'Storage Pool was updated.'
     MSG_DELETED = 'Storage Pool deleted successfully.'
     MSG_ALREADY_ABSENT = 'Storage Pool is already absent.'
-    MSG_MANDATORY_FIELD_MISSING = "Mandatory field was not informed: data.poolName"
+    MSG_MANDATORY_FIELD_MISSING = "Mandatory field was not informed:" \
+                                  " data.poolName (prior to API500) or data.name (API500 onwards)"
+    MSG_RESOURCE_NOT_FOUND = "The resource was not found. It is not possible to add it in API500 onwards." \
```

Comment:
`It can only be inventoried once through storage systems` feels somewhat weird to me at a first glance.

Could we perhaps change this message to something like:

* `The resource was not found. The addition of Storage Pools on API500 onwards is performed automatically through the Storage Systems.`

or

* `(...)found. Addition and removal of this resource on API500 onwards is managed by Storage Systems.`

Something along those lines?